signac/contrib/project.py

```diff
@@ -160,10 +160,9 @@ def __str__(self):
         return str(self.get_id())
 
     def __repr__(self):
-        return "{type}({{'project': '{id}', 'project_dir': '{rd}',"\
-               " 'workspace_dir': '{wd}'}})".format(
-                   type=self.__class__.__module__ + '.' + self.__class__.__name__,
-                   id=self.get_id(), rd=self.root_directory(), wd=self.workspace())
+        return "{type}.get_project('{root}')".format(
+                   type=self.__class__.__name__,
+                   root=self.root_directory())
```

Comment:
I feel like this is less useful than the previous behavior...

Reply:
Ok, I found the previous representation a bit too verbose and slightly redundant. But we can revert that change.

Reply:
Solution: we'll keep this `repr` as-is and add the other info to the `_repr_html_` method.